common/rpc/encryption/localStoreCertProvider.go

```diff
@@ -200,25 +274,12 @@ func (s *localStoreCertProvider) fetchCertificate(cachedCert **tls.Certificate,
 		return nil, errors.New("Cannot specify both certFile and certData properties")
 	}
 
-	s.RLock()
-	if *cachedCert != nil {
-		defer s.RUnlock()
-		return *cachedCert, nil
-	}
-
-	s.RUnlock()
-	s.Lock()
-	defer s.Unlock()
-
-	if *cachedCert != nil {
-		return *cachedCert, nil
-	}
-
 	var certBytes []byte
 	var keyBytes []byte
 	var err error
 
 	if certFile != "" {
+		s.logger.Info(fmt.Sprintf("loading certificate from file: %s", certFile))
```

Comment:
let's use a tag to specify dynamic data as opposed to placing dynamic values into the log message (which is typically supposed to be static)